Packs/Akamai_SIEM/ReleaseNotes/1_0_14.md

```diff
@@ -0,0 +1,14 @@
+
+<~XSIAM>
+#### Integrations
+
+##### Akamai WAF SIEM
+
+- Added the ***fetch-events*** command.
+
+#### Modeling Rules
+
+##### New: Akamai SIEM Modeling Rule
+
+Added support for Modeling Rules (Available from Cortex XSIAM 1.3.0).
```

Comment:
support for modeling rules? Is this standard? 
`Added the Akamai_WAF event modeling rules` or something like this